### PR TITLE
Update Helm Chart

### PR DESCRIPTION
**Pull request by mmattel**
This PR updates/improves the HC documentation.

* Harmonize file naming
* Remove comments not longer needed
* Text improvements
* Table improvements

### CODE REVIEW

File: modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc [hunk 1]
```diff
@@ -393,42 +377,42 @@ include::example$deployment/container/orchestration/email-notification-tab-3.yam
...
+These reasons add up and as a result, the responsibility for these generic secrets and their lifecycle is handed over to the operator who uses the Helm Charts. Any information necessary to use this security relevant data is provided by ownCloud via examples.
 
 // edit the yaml data shown at example$deployment/container/orchestration/
 
-:t_text: Secrets file containing all relevant secrets configurations.
+:t_text: Secrets file containing all relevant generic secrets configurations.
 
-* The following example shows what secrets need to look like and how they can be generated. The example assumes that the `secretRefs` are not changed . Each secret data entry holds a description on how to generate it or find the right value.
+* The following example shows how generic secrets need to look like and how they can be generated. The example assumes that the `secretRefs` are not changed . Each secret data entry holds a description on how to generate it or find the right value.
```
Comment (phil-davis) [inline modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc:390]:
```suggestion
* The following example shows what generic secrets need to look like and how they can be generated. The example assumes that the `secretRefs` are not changed . Each secret data entry holds a description on how to generate it or find the right value.
```

File: modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc [hunk 2]
```diff
@@ -437,49 +421,49 @@ ifdef::use_tab_3[]
...
 
-These reasons add up and as a result, the responsibility for these user management secrets and their lifecycle is handed over to the operator who uses the Helm Charts. Any information necessary to use the security relevant data is provided by ownCloud via examples.
+These reasons add up and as a result, the responsibility for these user management secrets and their lifecycle is handed over to the operator who uses the Helm Charts. Any information necessary to use this security relevant data is provided by ownCloud via examples.
 
 // edit the yaml data shown at example$deployment/container/orchestration/
 
 :t_text: Secrets file for the builtin user management.
 
-* The following example shows what user management secrets need to look like and how they can be generated. The example assumes that the `secretRefs` are not changed . Each secret data entry holds a description on how to generate it or find the right value.
+* The following example shows how user management secrets need to look like and how they can be generated. The example assumes that the `secretRefs` are not changed . Each secret data entry holds a description on how to generate it or find the right value.
```
Comment (phil-davis) [inline modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc:459]:
```suggestion
* The following example shows what user management secrets need to look like and how they can be generated. The example assumes that the `secretRefs` are not changed . Each secret data entry holds a description on how to generate it or find the right value.
```

File: modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc [hunk 3]
```diff
@@ -533,22 +517,22 @@ minikube kubectl apply -f builtin-user-mgmt-secrets.yaml
...
-These reasons add up and as a result, the responsibility for these user management secrets and their lifecycle is handed over to the operator who uses the Helm Charts. Any information necessary to use the security relevant data is provided by ownCloud via examples.
+These reasons add up and as a result, the responsibility for these user management secrets and their lifecycle is handed over to the operator who uses the Helm Charts. Any information necessary to use this security relevant data is provided by ownCloud via examples.
 
 // edit the yaml data shown at example$deployment/container/orchestration/
 
 :t_text: Secrets file for the external user management.
 
-* The following example shows what user management secrets need to look like and how they can be generated. The example assumes that the `secretRefs` are not changed . Each secret data entry holds a description on how to generate it or find the right value.
+* The following example shows how external user management secrets need to look like and how they can be generated. The example assumes that the `secretRefs` are not changed . Each secret data entry holds a description on how to generate it or find the right value.
```
Comment (phil-davis) [inline modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc:528]:
```suggestion
* The following example shows what external user management secrets need to look like and how they can be generated. The example assumes that the `secretRefs` are not changed . Each secret data entry holds a description on how to generate it or find the right value.
```

File: modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc [hunk 4]
```diff
@@ -651,14 +635,14 @@ helm install ocis ./charts/ocis --values values.yaml
 
 ==== Define Chart Access 
 
-. Get the minikube IP address:
+. Get, respectively check the minikube IP address:
```
Comment (phil-davis) [inline modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc:638]:
```suggestion
. Get, and check, the minikube IP address:
```

**Review by phil-davis — APPROVED** [review]

**Comment by mmattel** [comment]
Thanks for the review 👍

**Pull request merged**